src/GraphQL.Common/GraphQL.Common.csproj

```diff
@@ -12,6 +12,7 @@
 	</PropertyGroup>
 
 	<ItemGroup>
+		<PackageReference Include="Humanizer" Version="2.5.16" />
```

Comment:
can the extra dependency be ignored?

Reply:
It performs some conversions between the code property names and the schema field names, primarily things like camel-casing. You'd have to invite an implementation of that locally.